data/talkactions/scripts/create_item.lua

```diff
@@ -48,5 +54,5 @@ function onSay(player, words, param)
 		end
 		player:getPosition():sendMagicEffect(CONST_ME_MAGIC_GREEN)
 	end
-	return false
+	return true
```

Comment:
I think its a logical/aesthetics change, I mean we should return the boolean true by default if to signal we had found no obstacles in the code, otherwise return false to signal we faced some breaking (or not) obstacle ending the function call prematurely.

But as I have forgotten to change one more instance in line 4, one of the maintainers or I code revert this change. This also happens in more files.